Item.h

```diff
@@ -1,17 +1,18 @@
 #pragma once
-
 #include <string>
 
 struct Character;
 struct Item
 {
-    virtual ~Item() { }
+    Item();
```

Comment:
`Item` should not have a default constructor.
The existing constructor exists to initialize the member variables. 
- [ ] revise